pkg/crd/k8c.io/kubermatic.k8c.io_clusters.yaml

```diff
@@ -579,6 +583,7 @@ spec:
                               url:
                                 type: string
                             required:
+                              - annotations
```

Comment:
As it's required, what about an existing cluster migration ? Is there some mechanism to add it into exsiting clusters ?

Reply:
No automatic existing cluster migration for this, but needs to be done manually if required.

Reply:
Let's talk about it tomorrow on the sig-virt meeting, I think during cluster reconciliation we can add it automatically if not exists.

Reply:
As well we might want to add this field as optional.

Reply:
We could discover it from PVC, but let's see if it's worth. @sankalp-r please add this to the meeting agenda for tomorrow.

Reply:
as we agreed we leave this field as optional.